types/client.d.ts

```diff
@@ -41,6 +42,8 @@ declare namespace Client {
     tls?: TlsOptions | null;
     /** */
     maxRequestsPerClient?: number;
+
+    interceptors?: {Client: DispatchInterceptor[] | undefined}
```

Comment:
do you need to mention `undefined` explicitly if it's an optional parameter already?